controllers/operatorconfigmap_controller.go

```diff
@@ -329,11 +328,11 @@ func (c *ClusterVersionReconciler) createOrUpdate(obj client.Object, f controlle
 	return nil
 }
 
-func (c *ClusterVersionReconciler) own(obj client.Object) error {
-	return controllerutil.SetControllerReference(c.OperatorDeployment, obj, c.Client.Scheme())
+func (c *OperatorConfigMapReconciler) own(obj client.Object, operatorConfigMap *corev1.ConfigMap) error {
```

Comment:
The entire point of having an own util on the reconciler is to get an ownership ref by the input cr and not to take an owner as an argument. If you need to add a specialized owner you can use `SetControllerReference` instead. Please revert the change